exercises/2/webapp/src/components/NewsList.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="newslist_container">
+    <div>
+      <ul>
+        <li v-for="anews in newsSorted" :key="anews.id">
+          <NewsItem :news="anews" @update="update" @remove="remove" />
+        </li>
+      </ul>
+      <div v-if="isEmpty" class="emptyList">
+        The list is empty :(
+      </div>
+    </div>
+    <NewsForm :newTitle="newTitle" @create="create"/>
+    <button @click="reverse" id="reverseButton">
+      Reverse order
+    </button>
+  </div>
+</template>
+
+<script>
+import NewsItem from "./NewsItem";
```

Comment:
I see both single and double quotes used in the vue and js files. How is the linter configured?

Reply:
They apparently configured it to only show errors. I also prefer fixing those smaller things.
In any case, it's not a must for the homework.